contrib/scripts/ha/ha_make_slavedb.sh

```diff
@@ -238,7 +239,7 @@ function get_output_from_replica()
 	mkdir $output
 	
 	for replica_host in ${replica_hosts[@]}; do
-		scp_from_expect "$cubrid_user" "$server_password" $output $replica_host $output/$replica_host 
+		scp_from_expect "$cubrid_user" "$server_password" $output $replica_host $output/$replica_host
```

Comment:
an unrelated change?

Reply:
yes, unrelated change. once I modified this line but reverted it to the original.

Reply:
Why does it say that it has been changed although you reverted it?

Reply:
it shows the change because the space after "$output/$replica_host" is deleted unintendly.

Reply:
Ok, I misunderstood your comment. I thought you reverted unintended change already.

Reply:
So, please remove all the unintended changes.

Reply:
OK, I removed all the unintended changes